sound/soc/sof/ipc4-pcm.c

```diff
@@ -465,10 +465,7 @@ static int sof_ipc4_pcm_dai_link_fixup(struct snd_soc_pcm_runtime *rtd,
 	struct snd_interval *rate = hw_param_interval(params, SNDRV_PCM_HW_PARAM_RATE);
 	struct snd_mask *fmt = hw_param_mask(params, SNDRV_PCM_HW_PARAM_FORMAT);
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(component);
-	struct snd_soc_dai *cpu_dai = asoc_rtd_to_cpu(rtd, 0);
 	struct sof_ipc4_copier *ipc4_copier;
-	bool use_chain_dma = false;
```

Comment:
Commit message: you need to clarify why it's ok NOT to use 32-bits for the dai when the firmware uses 32-bits internally. This is really confusing.

Reply:
sure, updated. Thanks!